### ISSUE TITLE
Accessibility issue

### ISSUE THREAD

**Issue by andrushkin2** [ilya-kovshik/form-translate#24]
1. The cross icon should be a button with the SVG inside.
2. The button should have title and `aria-label` attributes `Remove phrase` or `Remove translation`

<img width="1319" alt="Screenshot 2022-05-05 at 13 15 28" src="https://user-images.githubusercontent.com/3307543/166904200-5234c6b3-9d1a-4416-953d-05d5ede47524.png">

**Comment by ilya-kovshik** [comment]
Fix has been pushed - [commit](https://github.com/ilya-kovshik/form-translate/commit/144b5b5b4fbfd9b7ccc497f7a18c66b990dc8a3c)

**Comment by andrushkin2** [comment]
@ikovshik 
Remove the`tabIndex="-1"`. The button should be focusable when navigating with keyboard.

<img width="1437" alt="Screenshot 2022-05-24 at 10 54 29" src="https://user-images.githubusercontent.com/3307543/169979158-c5af4c6b-f95b-46eb-8121-124d21acb961.png">

**Comment by ilya-kovshik** [comment]
Fix has been pushed - [commit](https://github.com/ilya-kovshik/form-translate/commit/11f0e4de0a60be667d57212223a5b37013d4b541)
